src/main/java/com/searchcode/app/jobs/IndexGitRepoJob.java

```diff
@@ -189,7 +189,7 @@ public boolean ignoreFile(String fileParent) {
 
     /**
      * Uses the inbuilt git
-     * TODO this method appears to leak memory like crazy... need to investigate
+     * TODO this method appears to leak memory like crazy... need to investigate. Ruzzz UPD.: Use method .close() of JGit objects?
```

Comment:
Agreed. I think from memory that the memory leak was actually due to other processes slowing this down. Will need to profile and check again.